### ISSUE TITLE
[Tooling] why is it taking so long for chip-tool interactive to come up in the ci?

### ISSUE THREAD

**Issue by cecille** [project-chip/connectedhomeip#31941]
### Reproduction steps / Feature

See https://github.com/project-chip/connectedhomeip/pull/31893#issuecomment-1927084736. yikes.

### Platform

other

### Type

CI tested

**Comment by bzbarsky-apple** [comment]
Just to be clear: right now we don't know whether it's "fails to come up withing 10s" or "fails to come up, period".

It's not 100% clear how to tell those cases apart short of somehow tracking the distribution of number of retries it takes in CI...

**Comment by cecille** [comment]
Yup, totally fair. @kliao-csa - do your scripts have any insight here?
